nixos/modules/hardware/ubertooth.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.ubertooth;
+
+  ubertoothPkg = pkgs.ubertooth.override {
+    udevGroup = cfg.group;
+  };
+in {
+  options.hardware.ubertooth = {
+    enable = mkEnableOption "Enable the Ubertooth software and its udev rules.";
+
+    group = mkOption {
```

Comment:
I'm not sure the group option provides much value, when users need to add themselves to the group manually anyway, which in turn would also create the group.

`users.groups.ubertooth.members = [ "me" ];`

A better option would IMO be one to specify users that will be added to the ubertooth group.

Reply:
On the one hand this is probably true, however, I have tried to adapt to other hardware modules. Most modules are just specifying a fixed group (e.g. `acpilight`, `bladeRF`, `brillo`). However, some modules are making this group configurable, e.g., `nitrokey`.

Specifying a list of users would be a third way in which the use of different modules differs even more. So I wouldn't find it so appropriate. Should I just configure a fixed group then?